librz/include/rz_cons.h

```diff
@@ -1070,6 +1071,20 @@ typedef struct rz_line_hist_t {
 	bool do_setup_match;
 } RzLineHistory;
 
+/**
```

Comment:
I think this whole structure is not even necessary here at this point and it could be hidden inside line.c or dietline.c, because the users of RzCons don't require any direct access to it, right?

Reply:
exactly right. I will hide it.